scripts/linters/pre_commit_linter.py

```diff
@@ -196,77 +197,71 @@ def _get_data(self, filepath, mode):
         return self._CACHE_DATA_DICT[key]
 
 
-def _lint_all_files(
-        js_filepaths, ts_filepaths, py_filepaths, html_filepaths,
-        css_filepaths, file_extensions_to_lint, verbose_mode_enabled=False):
+def _lint_all_files(file_extension_to_lint, verbose_mode_enabled=False):
     """Run all lint checks.
 
     Args:
-        js_filepaths: list(str). The list of js filepaths to be linted.
-        ts_filepaths: list(str). The list of ts filepaths to be linted.
-        py_filepaths: list(str). The list of python filepaths to be linted.
-        html_filepaths: list(str). The list of HTML filepaths to be linted.
-        css_filepaths: list(str). The list of CSS filepaths to be linted.
+        file_extension_to_lint: str. The file extension to be linted.
         verbose_mode_enabled: bool. True if verbose mode is enabled.
-        file_extensions_to_lint: list(str). The list of file extensions to be
-            linted.
 
     Returns:
         custom_linter: list. Custom lint checks.
         third_party_linter: list. Third party lint checks.
     """
     parent_dir = os.path.abspath(os.path.join(os.getcwd(), os.pardir))
-    config_path_for_css_in_html = os.path.join(
-        parent_dir, 'oppia', '.stylelintrc')
-    config_path_for_oppia_css = os.path.join(
-        parent_dir, 'oppia', 'core', 'templates', 'css', '.stylelintrc')
     custom_linters = []
     third_party_linters = []
 
-    js_ts_file_extension_type = 'js' in file_extensions_to_lint or (
-        'ts' in file_extensions_to_lint)
-    py_file_extension_type = 'py' in file_extensions_to_lint
-    html_file_extension_type = 'html' in file_extensions_to_lint
-    css_file_extension_type = 'css' in file_extensions_to_lint
-    other_file_extension_type = 'other' in file_extensions_to_lint
+    js_ts_file_extension_type = '.js' == file_extension_to_lint or (
```

Comment:
`file_extension_type_is_js_ts` or `file_extension_type_js_ts` or `extension_type_js_ts`.

This will make it more readable (Like `if file_extension_type_is_js_ts:` instead of `if js_ts_file_extension_type`)

Reply:
Done.